omnibus/files/private-chef-cookbooks/private-chef/libraries/preflight_checks.rb

```diff
@@ -0,0 +1,54 @@
+class PreflightValidationFailed < Exception
```

Comment:
I think it would be better to make these exceptions inherit from StandardError.  We try to avoid inheriting from Exception since it is the base of the exception hierarchy so it is dangerous to rescue it since it includes Interrupts as well.